posthog/settings/__init__.py

```diff
@@ -48,6 +48,7 @@
 INSTANCE_PREFERENCES = {"debug_queries": debug_queries, "disable_paid_fs": disable_paid_fs}
 
 SITE_URL: str = os.getenv("SITE_URL", "http://localhost:8000").rstrip("/")
+INSTANCE_TAG: str = os.getenv("INSTANCE_TAG", "default")
```

Comment:
Maybe `"none"` if there's no tag? That'd be a bit more intuitive to me than `"default"`